### PR TITLE
Completed first task

### PR DESCRIPTION
**Pull request by keda-vlad**

### CODE REVIEW

File: src/main/java/mate/academy/bookstore/BookstoreApplication.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class BookstoreApplication {
+    @Autowired
+    private BookService bookService;
```
Comment (Denis-Balako) [inline src/main/java/mate/academy/bookstore/BookstoreApplication.java]: Don't forget about dependency injection - provide this service through constructor and add @ Autowired above constructor.

File: src/main/java/mate/academy/bookstore/model/Book.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String author;
+    @Column(nullable = false, unique = true)
+    private String isbn;
+    @Column(nullable = false)
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
```
Comment (Denis-Balako) [inline src/main/java/mate/academy/bookstore/model/Book.java:27]: https://medium.com/@centizennationwide/mysql-naming-conventions-e3a6f6219efe
```suggestion
    @Column(name = "cover_image")
    private String coverImage;
```

File: src/main/java/mate/academy/bookstore/service/BookServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import java.util.List;
+import mate.academy.bookstore.model.Book;
+import mate.academy.bookstore.repository.BookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+    @Autowired
+    private BookRepository bookRepository;
```
Comment (Denis-Balako) [inline src/main/java/mate/academy/bookstore/service/BookServiceImpl.java]: Same as in BookstoreApplication about DI

**Review by Denis-Balako — CHANGES_REQUESTED** [review]
Good solution! Let's improve it 😄

**Review by Denis-Balako — APPROVED** [review]
LGTM!

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**